pkgs/development/libraries/tiledb/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchgit
+, cmake
+, zlib
+, lz4
+, bzip2
+, zstd
+, spdlog_0
+, tbb
+, openssl
+, boost
+, libpqxx
+, clang-tools
+, catch2
+, python
+, gtest
+}:
+
+stdenv.mkDerivation {
+  name = "tiledb";
+  version = "1.6.2";
+
+  src = fetchgit {
```

please ue fetchFromGitHub, this will pull down a tarball instead of the entire git history
```suggestion
  src = fetchFromGitHub {
    owner = "TileDB-Inc";
    repo = pname;
    rev = version;
    sha256 = "15i8rc2w3pnjz6chraf8nznk8p4mpgi4nx88v9jrg2s3hh092drb";
```

---

ok.

---

Done.